pick.lic

```diff
@@ -116,12 +116,16 @@ class LockPicker
         { name: 'pets', regex: /pets/, optional: true, description: 'Disarm boxes and place them in the pet box container.' },
         { name: 'count', regex: /\d+/, optional: true, description: 'How many pet boxes to make.' },
         { name: 'refill', regex: /refill/, optional: true, description: 'Refills your lockpick ring.' },
-        { name: 'source', regex: /(source=(\w+))/, optional: true, description: 'Container with boxes to pick.'}
+        { name: 'source', regex: /(source=(\w+))/, optional: true, description: 'Container with boxes to pick.'},
+        { name: 'force', regex: /force/i, optional: true, description: 'Force picking, ignoring any experience mindstate limits' },
+        { name: 'perception', regex: /perception/i, optional: true, description: 'Skip perception checks' }
       ]
     ]
 
     args = parse_args(arg_definitions, true)
-
+	
+    @skip_identify = args.perception
+    @force_repair = args.force
```

Comment:
`_repair` doesn't feel like the right name. What does this have to do with repairing?

Reply:
Corrected with new commit